### ISSUE TITLE
Allow the user to set a display name

### ISSUE THREAD

**Issue by nightsh** [tenders-exposed/elvis-ember#84]
...which is a name we'd use throughout all the interface, if set.

@georgiana-b, after you add this to the backend, can you test if it works with the frontend as well? It's implemented but we don't know if it's dead or alive.

**Comment by nightsh** [comment]
Blocked by tenders-exposed/elvis-backend#19

**Comment by zufanka** [comment]
@georgiana-b could you please add this issue on the new backend?

**Comment by victor-nitu** [comment]
Won't fix.
